### PR TITLE
1.2.0 hotfix

### PR DESCRIPTION
**Pull request by willbach**

### CODE REVIEW

File: android/app/src/main/java/com/lndr/PayPalModule.java [hunk 1]
```diff
@@ -72,13 +73,20 @@ public static String getConfigProperty(String key, Context context) {
...
         if (auth != null) {
           try {
             String authorization_code = auth.getAuthorizationCode();
             promise.resolve(authorization_code);
           } catch (Exception e) {
             promise.reject("Unknown error", e);
           }
+        } else if (confirm != null) {
+          try {
+            promise.resolve("Success");
```
Comment (arutun) [inline android/app/src/main/java/com/lndr/PayPalModule.java]: We should be passing the confirmation token/code here, not a human-readable string.

File: android/app/src/main/java/com/lndr/PayPalModule.java [hunk 2]
```diff
@@ -147,6 +155,8 @@ public void sendPayPalPayment(Double amount, String currencyCode, String payeeEm
         PayPalPayment.PAYMENT_INTENT_SALE
       );
 
+    payment.payeeEmail(payeeEmail);
```
Comment (arutun) [inline android/app/src/main/java/com/lndr/PayPalModule.java]: Can we do some parameter checking in this function prior to creating the PayPalPayment object to make sure the passed in parameters have lengths > 0 ?

File: packages/actions/index.ts [hunk 3]
```diff
@@ -1026,3 +1026,9 @@ const settleBilateral = async (user, bilateralSettlements, dispatch, getState) =
     }
   })
 }
+
+export const showPayPalSettlementError = (nickname: string) => {
+  return async (dispatch, _getState) => {
+    dispatch(displayError(settlementManagement.bilateral.error.generic(nickname)))
```
Comment (arutun) [inline packages/actions/index.ts]: I'm not getting a Toast error popup or any other message to the user.

File: packages/ui/components/paypal-settle-button/index.tsx [hunk 4]
```diff
@@ -91,7 +94,7 @@ class PayPalSettlementButton extends Component<Props, State> {
 
   async handlePayPalPayment() {
     try {
-      let cleanAmount = Number(this.props.displayAmount.replace(/[^0-9\.\,]/g, ''))
+      let cleanAmount = Number(this.props.displayAmount.replace(/[^0-9\.,]/g, ''))
```
Comment (arutun) [inline packages/ui/components/paypal-settle-button/index.tsx]: This did not fix the problem I was seeing so I don't know if it's necessary or not, probably...

File: android/app/src/main/java/com/lndr/PayPalModule.java (hunk 2 above)
Comment (willbach) [reply android/app/src/main/java/com/lndr/PayPalModule.java]: yes, and I would prefer to do the parameter checking on the JS level

File: packages/ui/components/paypal-settle-button/index.tsx (hunk 4 above)
Comment (willbach) [reply packages/ui/components/paypal-settle-button/index.tsx]: there isn't a difference between these two regular expressions

File: packages/actions/index.ts (hunk 3 above)
Comment (willbach) [reply packages/actions/index.ts]: I'll work on this

**Pull request merged**